1.9/usage/managing-services/rest-api.md

```diff
@@ -1,9 +1,29 @@
 ---
-post_title: Marathon REST API 
-nav_title: Marathon REST API 
+post_title: Marathon REST API
+nav_title: Marathon REST API
 menu_order: 40
 ---
 
+The Marathon API allows you to manage long-lived containerized services (apps and pods).
+
+The Marathon API is backed by the [Marathon component](/docs/1.9/overview/architecture/components/#marathon), which runs on the master nodes.
+
+One of these instances of Marathon is elected as leader, while the rest are hot backups in case of failure. All API requests must go through the Marathon leader. To enforce this, Admin Router proxies requests from any master node to the Marathon leader.
```

Comment:
"One of the instances of Marathon on a master node is elected..." -- I don't necessarily like my suggestion, but "these instances" is vague.